packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -555,9 +539,6 @@ export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
             case POOLS.curve_aave:
                 return 800e3;
             case POOLS.curve_PAX:
-            case POOLS.curve_y:
-            case POOLS.curve_BUSD:
-                return 850e3;
```

Comment:
Do we not want to keep the return value for `POOLS.curve_PAX`?

Reply:
rofl yes